### PR TITLE
Run unit tests using Jenkins

### PR DESCRIPTION
**Pull request by MonssafToukal**

### CODE REVIEW

**Comment by MonssafToukal** [comment]
@JSOBot runtests

File: Project.toml [hunk 1]
```diff
@@ -1,19 +1,19 @@
...
 uuid = "34c5aeac-e683-54a6-a0e9-6e0fdc586c50"
-version = "0.2.1"
+version = "0.2.2"
 
 [deps]
 BinaryProvider = "b99e7846-7c00-51b0-8f62-c81ae34c0232"
 Libdl = "8f399da3-3557-5675-b5ff-fb832c97cbdb"
 LinearAlgebra = "37e2e46d-f89d-539d-b4ee-838fcccc9c8e"
-METIS_jll = "d00139f3-1899-568f-a2f0-47f597d42d70"
+METIS4_jll = "40b5814e-7855-5c9f-99f7-a735ce3fdf8b"
```
Comment (dpo) [inline Project.toml:9]: Is it normal that this changed?

**Comment by dpo** [comment]
@MonssafToukal I’m surprised this kind of message doesn’t appear in the logs : https://github.com/JuliaSmoothOptimizers/HSL.jl/blob/main/test/test_ma57.jl#L32

File: Project.toml (hunk 1 above)
Comment (MonssafToukal) [reply Project.toml:9]: I don't think so, it shouldn't have changed. I merged the main branch before adding changes of my own.

Comment (dpo) [reply Project.toml:9]: But it’s already in main: https://github.com/JuliaSmoothOptimizers/HSL.jl/blob/main/Project.toml#L9

**Comment by MonssafToukal** [comment]
I will investigate, I think it's probably because the `tar.gz` are not found for some reason.

**Comment by dpo** [comment]
What’s in build.log ?

**Comment by MonssafToukal** [comment]
@JSOBot runtests

**Comment by MonssafToukal** [comment]
@dpo Now the problem with environment variables is fixed on the Jenkins' side of things.

**Comment by dpo** [comment]
Great! Now there seems to be an issue with LD_LIBRARY_PATH.

File: Project.toml (hunk 1 above)
Comment (amontoison) [reply Project.toml]: We did it together @dpo to be able to use `HSL.jl` with other packages that require METIS 5. We recompile METIS 4 with an other Artifact name : https://github.com/JuliaPackaging/Yggdrasil/pull/2755

**Comment by dpo** [comment]
@JSOBot runtests

**Comment by dpo** [comment]
I'm going to close this; I think I fixed it in #73.

**Pull request closed without merging**